### PR TITLE
Fix debugger.properties 

### PR DESCRIPTION
**Pull request by juliandescottes**
### Summary of Changes

* synchronization from MC (Bug 1334154)
* fixed localization notes for some context menu keys

cc @jasonLaster, @flodolo

### CODE REVIEW

File: assets/panel/debugger.properties [hunk 1]
```diff
@@ -166,9 +136,9 @@ sourceSearch.search.key=F
 # the search for re-searching the same search triggered from a sourceSearch
 sourceSearch.search.again.key=G
 
-# LOCALIZATION NOTE (sourceSearch1.resultsSummary): Shows a summary of
+# LOCALIZATION NOTE (sourceSearch.resultsSummary): Shows a summary of
 # the number of matches for autocomplete
-sourceSearch1.resultsSummary=%d instances of “%S”
+sourceSearch.resultsSummary=%d instances of “%S”
```
Comment (flodolo) [inline assets/panel/debugger.properties]: This change should be reverted.

Comment (juliandescottes) [inline assets/panel/debugger.properties]: Are you sure about this? The current string in mozilla central is sourceSearch.resultsSummary:

https://hg.mozilla.org/mozilla-central/file/tip/devtools/client/locales/en-US/debugger.properties#l129

(that's one of the things we changed in Bug 1334154)

Comment (flodolo) [inline assets/panel/debugger.properties]: Argh, forget this. I have no idea why, but I thought I was commenting on a different string (the one with %s vs %S). Too many tabs open I guess :-\

Comment (juliandescottes) [inline assets/panel/debugger.properties]: No worries :) thanks for reviews and sorry for the spam!

**Comment by jasonLaster** [comment]
Thanks for the help @juliandescottes!

**Pull request merged**